inc/taxonomies.php

```diff
@@ -443,29 +474,3 @@ function largo_first_headline_in_post_array($array) {
 
 	return $headline;
 }
-
```

Comment:
This PR has been updated: The `unregister_series_taxonomy` code has been placed into `largo_custom_taxonomies` where the series taxonomy is registered.